pages/docs/install/install-linux.md

```diff
@@ -42,6 +42,10 @@ note: The 'make install' is required to be run as root to get a properly install
 {% include asciicast.html source='install-singularity.js' uid='install-singularity' title='Install Singularity' author='vsochat@stanford.edu' %}
 
 
+### Prefix in special places
+
+If you build Singularity with a non-standard `--prefix` argument, please be sure to review the <a href="../admin-docs/admin-guide">admin guide</a> for details regarding the `--localstatedir` variable. This is especially important in environments utilizing shared filesystems.
```

Comment:
The link here, to work on github pages, I think might need to be just `/admin-guide`, see here --> http://singularity.lbl.gov/admin-guide